src/sass/components/_intro-story.scss

```diff
@@ -0,0 +1,79 @@
+.intro-story {
+  background-color: $gray-dark;
+  color: #fff;
+  text-align: center;
+  padding: 0;
+  height: 500px;
+  width: 100%;
+  @include breakpoints (small large){
+    height: 100vh;
+  }
+}
+
+.intro-story__background-image {
+    //Image for desktop has to be a very specific size otherwise it will tile or wrap
+    // onto the next slide (due to div width of react-slick carousel) - 1118 width works fine
+    background-image: url("/img/global/animations/3.1/flag_background_lg.png");
```

Comment:
@sarah-clements This is actually a case where the media query scoping I mentioned above (styling for smaller screens as the default and then scoping styles for larger screens in queries) becomes relevant. As it's currently set up, a small screen (or device) will only render the smaller image but still likely download both! Not what you want, especially if you're targeting mobile devices that are likely to have lower bandwidth!
Check out this for examples: https://timkadlec.com/2012/04/media-query-asset-downloading-results/